File: tests/java/metavar_equality_stmt.java
class foo {

void foo() {
    //ERROR:
    if (x > 2) {
        foo();
        bar();
    } else {
        foo();
        bar();
    }

    if (x > 2) {
        foo();
        bar();
    } else {
        foo();
    }
}


}

File: tests/patterns/rust/metavar_equality_stmt.rs
fn foo() {
    //ERROR:
    if (x > 2) {
        return 1;
    } else {
        return 1;
    }
    // nope, not here
    if (x > 2) {
        return 1;
    } else {
        return 2;
    }
}
